src/Microsoft.Health.Fhir.Core/Features/Operations/Import/CreateImportRequestHandler.cs

```diff
@@ -86,7 +86,7 @@ public async Task<CreateImportResponse> Handle(CreateImportRequest request, Canc
             {
                 TaskId = taskId,
                 TaskTypeId = ImportOrchestratorTask.ImportOrchestratorTaskId,
-                MaxRetryCount = _importTaskConfiguration.MaxRetryCount,
+                MaxRetryCount = -1, // Infinit retry for orchestrator task
```

Comment:
It is coordinator task that does not have any reason for failures of its own. In current implementation it fails pretty often because of huge amount of data passed to SQL.